README.md

```diff
@@ -48,9 +48,12 @@ _We also welcome also pull request contributions for new, useful commands!_
 
 On enterprise plans, Cody supports Anthropic Claude, Claude 2, and OpenAI GPT-4/3.5 models, with more coming soon.
 
-### Free usage
+### Usage
 
-Cody is currently in beta, and includes free LLM usage for individual users on both personal and work code. Fair use limits apply.
+- Cody Free: This version of Cody is available entirely free for all developers. It includes up to 500 autocomplete suggestions & 20 chat/command invocations per month.
+- Cody Pro: This is an expanded version of Cody for developers who want to use it every day, for either work or personal projects, with no usage limits. Cody Pro is available for $9/user/month.
```

Comment:
Note: it's better to just reference the pricing page instead of writing the exact price. It's more likely we will be consistent over the next months.

Reply:
I see your point. I made this change based on feedback that we weren't calling out Cody as a 'freemium' offering. I'll make this section more generic, FYI @taylorsperry